src/coreclr/vm/codeman.cpp

```diff
@@ -54,11 +54,16 @@ SPTR_IMPL(ReadyToRunJitManager, ExecutionManager, m_pReadyToRunJitManager);
 #endif
 
 #ifndef DACCESS_COMPILE
-Volatile<RangeSection *> ExecutionManager::m_CodeRangeList = NULL;
+Volatile<RangeSectionHandle *> ExecutionManager::m_RangeSectionHandleArray = nullptr;
+Volatile<int> ExecutionManager::m_LastUsedRSIndex = -1;
+Volatile<size_t> ExecutionManager::m_RangeSectionArraySize = 0;
```

Comment:
You've defined `m_RangeSectionArraySize` and `m_RangeSectionArrayCapacity` as LONG above in dacvars.h, this should be LONG too instead of size_t. Or replace LONG in dacvars.h with SIZE_T.